test/elementhandle.spec.js

```diff
@@ -336,14 +336,6 @@ module.exports.addTests = function({testRunner, expect}) {
       const content = await elementHandle.$$eval('.a', nodes => nodes.map(n => n.innerText));
       expect(content).toEqual(['a1-child-div', 'a2-child-div']);
     });
-
-    it('should throw in case of missing selector', async({page, server}) => {
```

Comment:
let's keep the test and make sure it doesn't throw in case of missing selector

Reply:
@aslushnikov - done, 10x!